test/expression/v2/eval_test.exs

```diff
@@ -61,11 +61,7 @@ defmodule Expression.V2.EvalTest do
     end
 
     test "functions with vars" do
-      assert 50 == eval("echo(10) * 5")
-    end
-
-    test "functions vars & properties" do
```

Comment:
something relevant to have removed this test?

Reply:
Echo isn’t a standard function and so I reworked these to not use echo. 

The properties on a function is already tested elsewhere by calling .match on the has_all_words function result.

Reply:
I see! thanks for clarifying